src/index.html

```diff
@@ -1,13 +1,49 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;900&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="Apple iMac"
+      >
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
```

Comment:
indent between this two elements is too big, check the design, also you can hardcode exact values for image because now it has wrong size, compare reference to the test
<img width="214" alt="Знімок екрана 2023-08-30 о 09 40 54" src="https://github.com/mate-academy/layout_product-cards/assets/106310796/84588520-c055-4282-aed1-fc635c6c2f23">